packages/data-sql/src/converter/index.ts

```diff
@@ -22,7 +23,25 @@ export const convertAbstractQueryToSqlStatement = (abstractQuery: AbstractQuery)
 			}
 		}),
 		from: abstractQuery.collection,
+		parameters: [],
 	};
 
+	const idGen = parameterIndexGenerator();
+
+	// The next functions look very similar.
+	// Depending on how the other conversions will look like, we can introduce a generic function for this.
```

Comment:
To make clear it is a temporary comment:
```suggestion
	// TODO:
	// The next functions look very similar.
	// Depending on how the other conversions will look like, we can introduce a generic function for this.
```